dali/kernels/slice/slice_cpu.h

```diff
@@ -30,6 +30,8 @@
 namespace dali {
 namespace kernels {
 
+static constexpr int kSliceCost = 2;  // compared to memcpy (heuristic)
```

Comment:
```suggestion
static constexpr const int kSliceCost = 2;  // compared to memcpy (heuristic)
```
...or face linker issues with clang